src/pmdas/bpf/modules/errno_helpers.c

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2020 Anton Protopopov
```

Comment:
> // Copied from the iovisor/bcc project at https://github.com/iovisor/bcc/blob/5f2a64ec101b2916558bb934c436d4a6d4ed157c/libbpf-tools/errno_helpers.c

I think it will be useful to add this for the commit message also...

Reply:
Sorry, missed this question earlier - @andreasgerstmayr as mentioned below, I wonder if we can vendor the libbpf-tools directory?